rest/src/routes.rs

```diff
@@ -58,6 +58,26 @@ impl<N: Network, B: BlockStorage<N>, P: ProgramStorage<N>> Server<N, B, P> {
             .and(with(self.ledger.clone()))
             .and_then(Self::get_transaction);
 
+        // GET /testnet3/transactions/mempool
```

Comment:
the comment doesn't match the path (it's plural)
```suggestion
        // GET /testnet3/transaction/mempool
```

Reply:
You may want to consider updating `/testnet3/transactions/{height}` singular as well, to make it consistent

Reply:
Good catch! For the `get_transaction` and `get_transactions`, we should definitely clarify, but there may be a collision in the warp parsing if we do the same path. I need to double check on this front